src/components/bitcoin_rate.js

```diff
@@ -1,12 +1,25 @@
-import React, { Component } from 'react';
+import React from 'react';
 
-class BitcoinRate extends Component {
-  render() {
-    return(
-      <div className="bitcoin-rate col-md-12">
+const BitcoinRate = ({postSelectedCurrency, postCurrencyVal, postCurrencyDesc, fontSizeTitle, fontSizePhrase}) => {
+
+let bitcoinPhrase = `Bitcoin Rate For ${postCurrencyDesc}, ${postSelectedCurrency} ${postCurrencyVal}`;
```

Comment:
on this instance, i needed bitcoinphrase to be dynamic because the content of bitcoinphrase variable will depend on the value of postSelectedCurrency value. that is why it bitcoinphrase is not constant type of variable